R/pkg/R/stats.R

```diff
@@ -134,9 +129,7 @@ setMethod("freqItems", signature(x = "SparkDataFrame", cols = "character"),
             collect(dataFrame(sct))
           })
 
-#' approxQuantile
-#'
-#' Calculates the approximate quantiles of a numerical column of a SparkDataFrame.
+#' approxQuantile - Calculates the approximate quantiles of a numerical column of a SparkDataFrame.
 #'
```

Comment:
I think we need to delete this line in between the `approxQuantile` line and the `The result of this algorithm` line. Otherwise the first line doesn't seem to show up in the rendered doc ?